src/app/pages/test/Test.js

```diff
@@ -1,3 +1,13 @@
-export default function Test({data}) {
-  return <div>{JSON.stringify(data)}</div>
+import { TextField, RadiosField } from 'common/forms'
+
+
+export default function Test({onSubmit}) {
+  return (
+    <form onSubmit={onSubmit}>
+      <TextField name="firstName" />
+      <TextField name="lastName" />
+      <RadiosField name="salutation" options={[{label: 'Mrs.', value: 'mrs'}, {label: 'Mr.', value: 'mr'}]}/>
```

Comment:
lets use spaces between object curly brackets for more readable code.  we can add rule to eslint config (https://eslint.org/docs/2.13.1/rules/object-curly-spacing)